be/src/exec/pipeline/pipeline_driver.cpp

```diff
@@ -484,8 +487,11 @@ const workgroup::WorkGroup* PipelineDriver::workgroup() const {
 }
 
 void PipelineDriver::set_workgroup(workgroup::WorkGroupPtr wg) {
-    this->_workgroup = std::move(wg);
-    this->_workgroup->incr_num_running_drivers();
+    _workgroup = std::move(wg);
+    if (_workgroup == nullptr) {
+        return;
```

Comment:
add a warning?

Reply:
No, `_workgroup` will nullptr value is legal. If the session variable `enabel_resource_group` is set to false, `_workgroup` is nullptr.